client/js/components/statement/listStatements/ExportModal.vue

```diff
@@ -0,0 +1,241 @@
+          {{ Translator.trans('docx.export.column.title') }}
+        </h3>
+        <dp-contextual-help
+          aria-labelledby="docxColumnTitles"
+          :text="Translator.trans('docx.export.column.title.hint')" />
+        <div class="grid grid-cols-5 gap-3 mt-1 mb-5">
+          <dp-input
+            v-for="(column, key) in docxColumns"
+            :key="key"
+            :id="key"
+            type="text"
+            :data-cy="column.defaultValue"
+            :placeholder="Translator.trans(column.placeholder)"
+            :width="column.width"
+            v-model="column.title"
+            @input="handleInput" />
+        </div>
+      </section>
+
+      <div class="flex justify-end">
+        <dp-button-row
+          primary
+          secondary
+          :primary-text="Translator.trans('export.statements')"
+          :secondary-text="Translator.trans('abort')"
+          @primary-action="handleExport"
+          @secondary-action="handleAbort" />
+      </div>
+    </dp-modal>
+  </div>
+</template>
+
+<script>
+import {
+  DpButtonRow,
+  DpContextualHelp,
+  DpInput,
+  DpModal,
+  DpRadio,
+  sessionStorageMixin
+} from '@demos-europe/demosplan-ui'
+
+export default {
+  name: 'ExportModal',
+
+  components: {
+    DpButtonRow,
+    DpContextualHelp,
+    DpInput,
+    DpModal,
+    DpRadio
+  },
+
+  mixins: [sessionStorageMixin],
+
+  data () {
+    return {
+      active: 'docx',
+      docxColumns: {
+        left: {
+          width: 'col-span-1',
+          dataCy: 'exportModal:input:colLeft',
+          placeholder: Translator.trans('segments.export.segment.id'),
+          title: null
+        },
+        middle: {
+          width: 'col-span-2',
+          dataCy: 'exportModal:input:colMiddle',
+          placeholder: Translator.trans('segments.export.statement.label'),
+          title: null
+        },
+        right: {
+          width: 'col-span-2',
+          dataCy: 'exportModal:input:colRight',
+          placeholder: Translator.trans('segment.recommendation'),
+          title: null
+        }
+      },
+      exportTypes: {
+        docx: {
+          label: 'export.statements.docx',
+          uploadPath: 'dplan_statement_segments_export',
+          dataCy: 'exportModal:export:docx'
+        },
+        zip: {
+          label: 'export.statements.zip',
+          uploadPath: 'dplan_statement_segments_export_packaged',
+          dataCy: 'exportModal:export:zip'
+        },
+        xlsx: {
+          label: 'export.statements.xlsx',
+          uploadPath: 'dplan_statement_xls_export',
+          dataCy: 'exportModal:export:xlsx'
+        }
+      }
+    }
+  },
+
+  methods: {
+    applyDefaultPlaceholdersForEmptyNames () {
+      Object.keys(this.docxColumns).forEach(key => {
+        if (!this.docxColumns[key].title) {
+          this.docxColumns[key].title = null
+          this.docxColumns[key].placeholder = this.getDefaultPlaceholderByKey(key)
+        }
+      })
+    },
+
```

Comment:
I suggest to remove this method here, the @input="handleInput" in the Template and the handleInput Method.
You should not have to change the Placeholder.